src/resolvers/myndlaResolvers.ts

```diff
@@ -6,7 +6,11 @@
  *
  */
 
-import { fetchConfig, fetchExamLockStatus } from '../api/myndlaApi';
+import {
+  fetchConfig,
+  fetchExamLockStatus,
+  fetchCategories,
```

Comment:
Sjans for at denne ikke brukes?

Reply:
Ganske høy viser det seg. Har vi andre linte-regler her?

Reply:
Det hadde ikke overrasket meg nevneverdig, hvertfall